src/main/react-front-end/src/views/Queue/QueueComponent.js

```diff
@@ -82,16 +82,17 @@ class QueueComponent extends Component {
 	}
 
 	update = () => {
-		let jobIds = this.state.responsesToDisplay.filter(job => job.status !== "complete" && job.status !== "failed" && job.status !== "removed")
+		let jobIds = this.state.responsesToDisplay.filter(job => !["complete", "failed", "removed", "cancelled"].includes(job.status))
```

Comment:
We can only check if job status includes processing instead of contradiction condition.

Reply:
Done